packages/core/src/actions/playback/getMetrics.ts

```diff
@@ -0,0 +1,9 @@
+import { GetMetricsArgs, LivepeerProvider, Metrics } from '../../types';
+import { getLivepeerProvider } from '../providers';
+
+export function getMetrics<
```

Comment:
This should probably be `getAssetMetrics` or maybe `getViewershipMetrics` (ended up commenting this on changelog and here). If you go with the latter, it should be consistent with the React hook as well replacing `get` with `use`

Reply:
Agreed!